docs/source/user/faq.rst

```diff
@@ -74,6 +74,13 @@ variables.  On a 32-bit machine, you may sometimes need the magnitude of
 ``np.int64`` (for example ``np.int64(0)`` instead of ``0``).  It will
 propagate to all computations involving those variables.
 
+How can I tell if ``parallel=True`` worked?
```

Comment:
Can you also link to this FAQ entry from the parallel accelerator user docs page?